docs/nodes/light-node.mdx

```diff
@@ -925,7 +925,7 @@ celestia light init --p2p.network blockspacerace
 You should see output like:
 
 ```output
-$ celestia light init
+$ celestia light init --p2p.network blockspacerace
```

Comment:
do we even need bsr anymore

Reply:
😂 no it's there as a placeholder for mainnet tbh, for the tabs. i'll make sure it's gone by then